lms/templates/ccx/schedule.html

```diff
@@ -29,80 +29,88 @@
   <div id="new-ccx-schedule"></div>
 </div>
 
-<section id="enter-date-modal" class="modal" aria-hidden="true">
-  <div class="inner-wrapper" role="dialog">
-    <button class="close-modal">
-      <i class="icon fa fa-remove"></i>
+<section id="enter-date-modal" class="modal"
+         tabindex="1" role="dialog" aria-labelledby="ccx_schedule_set_date_heading">
+  <div class="inner-wrapper">
+    <button class="close-modal" tabindex="2">
+      <i class="icon fa fa-remove" aria-hidden="true"></i>
       <span class="sr">
         ${_("Close")}
       </span>
     </button>
     <header>
-      <h2></h2>
+      <h2 tabindex="2" id="ccx_schedule_set_date_heading"></h2>
     </header>
     <form role="form">
       <div class="field datepair">
-        <label></label>
-        <input placeholder="Date" class="date" type="text" name="date"/ size="11">
-        <input placeholder="Time" class="time" type="text" name="time"/ size="6">
+        <input placeholder="Date" class="date" type="text" name="date"
+               id="date" size="11" tabindex="2" aria-label="${_('Enter date format yyyy-mm-dd')}">
```

Comment:
I like that you added screen reader text here to help with formatting. However, screen readers don't read yyyy-mm-dd very well as its not a word. In fact, it was unintelligible. I would recommend spelling it out "four digit year dash two digit month dash to digit day". Also should be internationalized.